README.md

```diff
@@ -65,13 +65,45 @@ npm run test
 
 ### Mock Server for Client Tests
 
-The `algod_client`, `indexer_client`, and `kmd_client` packages use a Docker-based mock server for deterministic API testing. By default, running tests will automatically start the mock server container.
+The `algod_client`, `indexer_client`, and `kmd_client` packages use a mock server for deterministic API testing against pre-recorded HAR files. The mock server is managed externally (not by the test framework).
 
-To use an external mock server (e.g., for local development):
+**In CI:** Mock servers are automatically started via the [algokit-polytest](https://github.com/algorandfoundation/algokit-polytest) setup.
+
+**Local development:**
+
+1. Clone algokit-polytest and start the mock servers:
```

Comment:
Wouldn't you have the polytest repo locally when after running `npm run polytest:validate-algod`. Maybe it's better to document the flow using `npm run` commands.